src/foundry/common/types.d.mts

```diff
@@ -275,9 +297,11 @@ declare global {
     /** Optional (reserved) modifiers */
     optionalModifiers: string[];
 
+    // TODO should this have the same type as KeybindingActionConfig.onDown?
     /** The handler that executes onDown */
     onDown: Function;
 
+    // TODO should this have the same type as KeybindingActionConfig.onUp?
     /** The handler that executes onUp */
     onUp: Function;
```

Comment:
This, of course, is a holdover but `Function` is a pretty broad type. Consider finding usages of `onDown`/`onUp` and see how it's called and then make it a more specific type.

Reply:
Confirmed the type and updated it accordingly.